include/lmms_math.h

```diff
@@ -352,6 +353,28 @@ static inline int numDigitsAsInt(float f)
 	return digits;
 }
 
+template <typename T>
```

Comment:
Do we want to limit `T` to certain types, or just let the compiler error if we try to use a type that isn't defined with any mathematical operators?

Reply:
I'd prefer the compiler error because that way the class gives more flexibility.